docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -3106,11 +3107,17 @@ for all column based noise include:
     - Missing data, nicknames, fake names, phonetic, OCR, typographic
     -
   * - Middle Initial
-    - Census, Household Surveys, WIC, Taxes (both), SSA
+    - Census, Household Surveys, WIC, Taxes (both)
     - 0.01
     - 0.1
     - Missing data, phonetic, OCR, typographic
     -
+  * - Middle Name
+    - SSA
+    - 0.01
+    - 0.1
+    - Missing data, nicknames, fake names, phonetic, OCR, typographic
+    - Use the list of fake first names for middle names as well
```

Comment:
@zmbc and @pletale:

@albrja [asked in Slack](https://ihme.slack.com/archives/C02KUQ9LX32/p1690914431727189) whether we wanted to apply all the noise types from first name to middle name as well, namely nicknames and fake names. I thought this made sense, so I included it here. Running it by the two of you since you already approved the PR without the two additional noise types. Any objections?

Reply:
Sounds good to me!